packages/web/src/pages/chat-page/components/ChatComposer.tsx

```diff
@@ -92,7 +93,8 @@ export const ChatComposer = (props: ChatComposerProps) => {
 
   const ref = useRef<HTMLTextAreaElement>(null)
   const chatIdRef = useRef(chatId)
-  const isBlast = chatId === ChatBlastAudience.FOLLOWERS
+  const { audience, audienceContentType, audienceContentId, isBlast } =
```

Comment:
Fwiw, this is fairly implicit - I think it's fine but it's worth noting there's nothing _technically_ saying that blasts have to follow this convention. It's possible that what we want is to have the `sendMessage` action used for both, and select the chat object inside the saga to determine if it's a blast or not and what the audience is?

Reply:
dude... brilliant.. why was i making life so hard for myself